File: currency_adjusted_roi/r_X.R
r_X <- function(r_Y, e_XY, e_XY_prime) {
  R_XY <- (e_XY_prime / e_XY) - 1
  r_X <- (r_Y - R_XY) / (1 + R_XY)
  r_X
}

File: currency_adjusted_roi/r_X.py
def r_X(r_Y, e_XY, e_XY_prime):
    R_XY = (e_XY_prime / e_XY) - 1
    r_X_value = (r_Y - R_XY) / (1 + R_XY)
    return r_X_value